src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -404,11 +404,11 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
               onClick={() => {
                 logEvent({
                   category: 'Downloads',
-                  action: `Release page all files downloads.title}, Release: ${release.title}, File: All files`,
+                  action: `Release page all files downloads.title, Release: ${release.title}, File: All files`,
```

Comment:
Probably would suggest changing this to 'Release page all files download. ....'

Would suggest checking with Laura / Cam.

Reply:
Checked &  changing this might be more effort than it's worth since they'd have to review if that introduces any breaking changes.